app/admin.js

```diff
@@ -34,6 +35,30 @@ importKeys.addArgument(
   }
 );
 
+const signCommand = subparsers.addParser('sign', { addHelp: true });
+signCommand.addArgument(
+  ['file'],
+  {
+    action: 'store',
+    help: 'path to the recovery request JSON file'
+  }
+);
+signCommand.addArgument(
+  ['key'],
+  {
+    action: 'store',
+    required: false, // can be typed during the signing process to avoid leaving the xprv in the shell history
+    help: 'private key to sign the transaction with'
+  }
+);
+signCommand.addArgument(
+  ['--confirm'],
```

Comment:
Is this just for automated signing? Should we allow automated signing?

Reply:
We allowed it in version 1 per Keyternal's request. It's also useful for unit tests.